internals/models.py

```diff
@@ -885,6 +885,46 @@ def getSortingMilestone(feature):
 
     return allowed_feature_list
 
+  @classmethod
+  def get_in_milestone(
+      self, version=None, show_unlisted=False, milestone=None):
+    if milestone == None:
+      return None
+
+    logging.info('Getting chronological feature list in milestone %d', milestone)
+    q = Feature.query()
+    q = q.order(Feature.name)
+    q = q.filter(Feature.shipped_milestone == milestone)
+    desktop_shipping_features = q.fetch(None)
+
+    # Features with an android shipping milestone but no desktop milestone.
+    q = Feature.query()
+    q = q.order(Feature.name)
+    q = q.filter(Feature.shipped_android_milestone == milestone)
+    q = q.filter(Feature.shipped_milestone == None)
+    android_only_shipping_features = q.fetch(None)
+
```

Comment:
@jrobbins, Features with desktop or android milestones that are same as the queried milestone, will be returned by the API.

Reply:
Looks good.